src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/MemberRelationshipService.cs

```diff
@@ -117,36 +141,30 @@ protected virtual void SetRelationship(MemberRelationship source, MemberRelation
                 throw new ArgumentException(SR.Format(SR.MemberRelationshipService_RelationshipNotSupported, sourceName, source.Member.Name, relName, relationship.Member.Name));
             }
 
-            if (_relationships == null)
-            {
-                _relationships = new Dictionary<RelationshipEntry, RelationshipEntry>();
-            }
```

Comment:
Why did you delete this?

Reply:
Because it is never reassigned after being initialized as a field. So after I made it `readonly` this field can never be null

Reply:
Then why is the check for null earlier at https://github.com/dotnet/corefx/pull/30159/files/bdc9a0273fa7b6de797c81fce0c9f8e204d696b4#diff-8ef5408d694347692e1f4b2865cbcdbbR114 still needed?

Reply:
It isn't! I've removed it